NuKeeper/Engine/PackageUpdateSelection.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NuKeeper.Configuration;
+using NuKeeper.RepositoryInspection;
+
+namespace NuKeeper.Engine
+{
+    public class PackageUpdateSelection : IPackageUpdateSelection
+    {
+        private readonly RepositoryModeSettings _settings;
+
+        public PackageUpdateSelection(RepositoryModeSettings settings)
```

Comment:
Yeah I'm kinda unhappy with the way it's all set up now - so that's another reason to not pass it round everywhere :P